### ISSUE TITLE
Modules mei.header and mei.frbr are not independent

### ISSUE THREAD

**Issue by riedde** [music-encoding/music-encoding#1388]
It has been articulated several times that the modules `header` and `frbr` are intervowen, e.g., [here](https://github.com/music-encoding/music-encoding/issues/1078#issuecomment-1430155989).

Problem in short: `<physDesc>` is part of `MEI.header` but only occurs as content of `<item>` which is part of `MEI.frbr`. If `MEI.frbr` is deactivated `<physDesc>` is defined but not allowed anywhere.

I would propose to move `<physDesc>` from the module `MEI.header` to `MEI.frbr`. For this I ask for comments.
(relates partly to https://github.com/music-encoding/music-encoding/issues/1078)

Other ideas I discarded:
* declaring `MEI.frbr` as core schema
* integrating `MEI.frbr` into `MEI.header`

**Comment by ahankinson** [comment]
How does this address the separability of FRBR from the metadata? If we want to use `<physDesc>` without `<item>`, how would moving it to FRBR help with that?

**Comment by pe-ro** [comment]
I agree with @ahankinson  here.  What we should be working toward is a fully functioning non-FRBR solution for those who want/need it while still allowing a FRBR approach as well.  The pull requests I made a couple of months back were an attempt to address this.  However, I can see how multiple pull requests muddies the water.  I hope to replace the multiple pull requests with a single, comprehensive one soon.

**Comment by riedde** [comment]
> How does this address the separability of FRBR from the metadata? If we want to use `<physDesc>` without `<item>`, how would moving it to FRBR help with that?

The reference was misleading. I didn't meant that this will relsolve the issue but we should keep this in mide because it can have side effects.

> What we should be working toward is a fully functioning non-FRBR solution for those who want/need it while still allowing a FRBR approach as well.

Fixing module independency and encoding physDesc without frbr are different issues for me but if both can be fixed together I'm fine with it.

> The pull requests I made a couple of months back were an attempt to address this. However, I can see how multiple pull requests muddies the water. I hope to replace the multiple pull requests with a single, comprehensive one soon.

Okay, then I don't really have an overview right now. I just wanted to make sure that this problem is not forgotten. Could you refer to this issue when the merged PR is opened? Thanks!

**Comment by ahankinson** [comment]
I'm sorry if my first message came off a bit short. It was actually a genuine question: How does moving `<physDesc>` to the FRBR module fix the problem that the FRBR module and the Header module are intertwined?

Moving `<physDesc>` to FRBR would seem to completely cancel the request in #1078, since it would make physical description of a source even more difficult without also using the FRBR module. 

I see the `MEI.header` module as a sort of `MEI.shared`, but only for metadata. That is, `MEI.header` and `MEI.frbr` are not peers, but that FRBR should depend on `MEI.header` for descriptive functionality that is shared. `MEI.frbr` should extend MEI to add FRBR-specific functions, but it shouldn't be the only way of doing source description available.

At the moment, the problem is that the dependencies are circular -- `MEI.frbr` depends on `MEI.header`, but `MEI.header` ALSO depends on `MEI.frbr`, since you cannot access `<physDesc>` without using `<item>`. If we make `<physDesc>` available on `<source>` (again), then we remove the hard dependency that `<physDesc>` is only available through the FRBR functions.

**Comment by riedde** [comment]
Thank you for this insight @ahankinson ! I think I got your point. 

> Moving <physDesc> to FRBR would seem to completely cancel the request in https://github.com/music-encoding/music-encoding/issues/1078, since it would make physical description of a source even more difficult without also using the FRBR module.

Now, I see that my proposal would be more complicated than also allowing `<physDesc>` in a `MEI.header` element.

But I'm not sure if [`<source>`](https://music-encoding.org/guidelines/dev/elements/source.html#attributes_compact_tab) is the best place, because it is defined to keep »bibliographic description«. But this was already adressed in #1078 and we don't have to duplicate this discussion here.

___________________________________________
@ahankinson just a side comment on
> How does moving `<physDesc>` to the FRBR module fix the problem that the FRBR module and the Header module are intertwined?

In your last paragraph you wrote »At the moment, the problem is that the dependencies are circular«. This circular dependency would be eliminated with moving the element and this was the only thing I had in mind opening this issue. (Idea: Elements only available when using `MEI.frbr` could also be part of it.)

**Comment by ahankinson** [comment]
> But I'm not sure if [`<source>`](https://music-encoding.org/guidelines/dev/elements/source.html#attributes_compact_tab) is the best place, because it is defined to keep »bibliographic description«

Physical description is very much part of bibliographic description. Size, extent, condition, carrier, and many other qualities relating to the physical object have been part of the bibliographic description process since the beginning.

**Comment by pe-ro** [comment]
Addressed in PR #1367

**Comment by bwbohl** [comment]
Is this issue resolved after merging #1366 and #1367?

**Comment by riedde** [comment]
Yes, since #1366 is merged, the `<physDesc>` element ist also available if the `MEI.frbr` module is not present. (Only `MEI.header` and `MEI.shared` are required, but these are part of the core modules, I think.)
